src/System.IO.Pipelines/PipelineConnectionExtensions.cs

```diff
@@ -88,7 +87,7 @@ public static Task CopyToEndAsync(this IPipeReader input, Stream stream)
                 input.Complete(ex);
```

Comment:
Shouldn't AsPipelineWriter() call this method instead of CopyToAsync() so exceptions are handled?